datasetviewer/tests/main_view_presenter_test.py

```diff
@@ -85,10 +85,10 @@ def test_create_default_plot(self):
         main_view_presenter.subscribe_preview_presenter(self.mock_preview_presenter)
         main_view_presenter.set_dict(self.fake_dict)
 
-        # Create a fake dataset and place it in the MainViewPresenter
+        # Instruct the MainViewPresenter to generate the default plot from one of the elements in the data dictionary
```

Comment:
On line 85, is it really necessary to bring the preview_presenter into play? If you don't need to set it, then don't.